src/initialState.js

```diff
@@ -4,12 +4,38 @@ const initialState = {
   projects: [
     {
       id: uuid(),
-      name: 'Sample Project',
+      name: 'Versus Todo List',
       tasks: {
-        count: 0,
-        todo: 0,
-        inProgress: 0,
-        list: []
+        count: 100,
+        todo: 40,
+        inProgress: 15,
+        complete: 45,
```

Comment:
I matched the design page's values, and assumed starting from count 0 would be the correct procedure, asking questions here would have been helpful from my part.  I can easily adjust / fix it with a quick pull request.  Sorry about that.